meetings/agenda-2018-02-16.md

```diff
@@ -0,0 +1,100 @@
+# ECMA 402 meeting agenda
+
+This is an initial agenda document; please add additional agenda items as appropriate through PRs to this repository.
+
+February 16th, 2018, 17:15 UTC
+
+Contact Daniel Ehrenberg (littledan@igalia.com) for the link to the Google Hangouts VC.
+
+1. TC39 summary
+    1. ECMA 402 2018 edition feature set finalized
+        1. [formatToParts](https://github.com/tc39/ecma402/issues/47)
+        1. [PluralRules](https://github.com/tc39/proposal-intl-plural-rules)
+        1. [hourCycle](https://github.com/tc39/ecma402/pull/135)
+    1. Features reached Stage 3
+        1. [ListFormat](https://github.com/tc39/proposal-intl-list-format)
+        1. [RelativeTimeFormat](https://github.com/tc39/proposal-intl-relative-time)
+        1. [Locale](https://github.com/tc39/proposal-intl-locale)
+    1. [lb option added to Segmenter](https://github.com/tc39/proposal-intl-segmenter/pull/24); [renamed to lineBreakStyle](https://github.com/tc39/proposal-intl-segmenter/pull/25)
+    1. [Settled on numeric: always/auto](https://github.com/tc39/proposal-intl-relative-time/pull/60)
+    1. [Explained the existence of this call; well-received](https://github.com/tc39/tc39-notes/blob/master/es8/2018-01/jan-23.md#4-ecma402-status-updates)
+    1. Non-members who contribute to ECMA 402 (either PRs or in this call): Sign [this form](https://tc39.github.io/agreements/contributor/) to license contributions to Ecma
+1. Questions/issues with existing advanced proposals and APIs
+    1. [Normative: Add calendar and numberingSystem options](https://github.com/tc39/ecma402/pull/175)
+        - Should we do validation which throws for these? Is there an ICU API?
+    1. [Segmenter: Should we add all the options (lw, ss), or just lb?](https://github.com/tc39/proposal-intl-segmenter/issues/23)
+        - At the last meeting, it seemed like Steven favorited this change but Jungshik was hesitant
+        - tiebreaker: stay minimal?
+    1. Locale:
+        1. [Consider performing complete Unicode extension canonicalization per RFC6067](https://github.com/tc39/proposal-intl-locale/issues/14)
+            - Namely, should we sort and deduplicate all of the extensions (not just the recognized ones)?
+            - If there are multiple -u- sections, appropriate to silently ignore following ones?
```

Comment:
Multiple -u- sections are not possible. https://tc39.github.io/ecma402/#sec-isstructurallyvalidlanguagetag
> - does not include duplicate singleton subtags.

Reply:
Oh, silly me, removed that line.